### DISCUSSION TITLE
Deployment of contract using the .env variables is giving error

### DISCUSSION THREAD

**Discussion by ikunal-04** [Cyfrin/foundry-full-course-cu#567] [Q&A] [answered]
So, my source .env command is working perfectly fine and I'm able to see the values of variables using the bash command line but when I'm trying to deploy the contract using those variables it is giving me this error:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/112847288/1b2fa534-c305-4f40-a8fc-2ef7e26727f2)

**Comment by shikhar229169** [comment, marked as answer]
Problem is here: 
 --rpc-url $RPC-URL

You put '-' instead of underscore.
You need to type: --rpc-url $RPC_URL.

**Reply by ikunal-04** [reply]
oh okay got it 👍 thank you
